app/MainHie.hs

```diff
@@ -51,7 +56,7 @@ plugins includeExamples = pluginDescToIdePlugins allPlugins
       , baseDescriptor        "base"
       , brittanyDescriptor    "brittany"
       , buildPluginDescriptor "build"
-      , ghcmodDescriptor      "ghcmod"
+      -- , ghcmodDescriptor      "ghcmod"
```

Comment:
delete this line.

Reply:
Done in https://github.com/mpickering/haskell-ide-engine/pull/14